src/skins/vector/css/themes/contrastlight.scss

```diff
@@ -0,0 +1,4 @@
+@import "_base.scss";
+@import "_dark.scss";
```

Comment:
I guess this explains why redacted messages appear to come from the dark theme! Perhaps this should be _light ?